unlock-app/src/components/Unlock.js

```diff
@@ -1,69 +0,0 @@
-import PropTypes from 'prop-types'
```

Comment:
Is this moved to somewhere else? Please clarify ;)

Reply:
Yes in /pages and withConfig